app/permissions/decidim/vocdoni/admin/permissions.rb

```diff
@@ -51,9 +58,35 @@ def current_vocdoni_wallet
           @current_vocdoni_wallet ||= Decidim::Vocdoni::Wallet.find_by(decidim_organization_id: user.organization.id)
         end
 
+        def census_status
+          CsvCensus::Status.new(election)&.name
+        end
+
         def allow_if_not_blocked
           toggle_allow(election && !election.blocked?)
         end
+
+        def allow_question_step
+          return if election.blank?
+
+          toggle_allow(election.present?)
```

Comment:
We should implement the new meths here: 
```suggestion
          toggle_allow(election.ready_for_questions_form?)
```